modules/search/instant-search/index.jsx

```diff
@@ -14,6 +14,11 @@ import { getThemeOptions } from './lib/dom';
 import { SERVER_OBJECT_NAME } from './lib/constants';
 import { initializeTracks, identifySite } from './lib/tracks';
 
+// Must be the first import
```

Comment:
Is it worth us removing this for the moment if it's not compatible with the current React Devtools?

https://marvinh.dev/blog/state-of-devtools/

Reply:
I'll go ahead and remove this before merging.